src/tomahawk/TomahawkWindow.cpp

```diff
@@ -851,7 +852,7 @@ TomahawkWindow::audioStateChanged( AudioState newState, AudioState oldState )
     Q_UNUSED(newState);
 #else
     updatePreview();
-        
+
```

Comment:
non-related change, but KDevelop removed the whitespace there and as it anyways was wrong I left it in the commit :)